src/makeInfinityAdder.js

```diff
@@ -5,6 +5,22 @@
  */
 function makeInfinityAdder() {
   // write code here
+  let sum = 0;
+
+  function adder(num) {
+    if (num === undefined) {
```

Comment:
Consider using `typeof num === 'undefined'` instead of `num === undefined` to avoid potential issues with variables that might be set to `undefined` explicitly.